src/azure-cli/azure/cli/command_modules/profile/_help.py

```diff
@@ -95,9 +138,23 @@
     - name: Get an access token to use with MS Graph API
       text: >
         az account get-access-token --resource-type ms-graph
+    - name: >
+        Get an access token for a particular resource.  For a list of Virtual Network available service endpoints,
```

Comment:
You can't get access token for ARM resources, you can only get access token for ARM. This word `resource` here is a concept of AAD/Graph, not that of ARM.